pyuploadcare/api_resources.py

```diff
@@ -231,24 +231,76 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
-        """Creates File copy
+    def local_copy(self, effects=None, store=None):
```

Comment:
that's more in line with the code in WC, while @homm's idea might be better aligned with lib's history. Let's go with copy_to_remote and copy_to_local.